docs/source/DeveloperDocumentation/VUDRM-token.md

```diff
@@ -15,6 +15,187 @@ VUDRM tokens should be generated using the [VUDRM token API](#vudrm-token-api).
+
+### Online renewable licenses in Widevine
+
+In order to have online licenses automatically renew when using Widevine you must first ensure the policy your VUDRM token is using contains `can_renew` set to `true`. With this value set Widevine will try to renew the license. This renewal will happen every 1800 seconds, although we can configure value so that license requests are more/less frequent based on your business needs. Although new licenses will be fetched at regular intervals, this interval is not based on when a license will expire and as such it is required that the license is valid for longer than the length of the interval between renewals. Below is an example policy for a renewable license that is only valid for 45 minutes but will renew ever 30 minutes.
+
+```JSON
+{
+    "can_renew": true,
+    "rental_duration_seconds": 2700
+}
+```
+
+### Match content ID
+If `content_id` has been set and `match_content_id` has been set to `true`, when a license request is made the content ID in the license request and the content ID in the VUDRM token will be compared. If they **are** the same a license will be served as normal; if they are **not** the same the license request will be denied. If `match_content_id` has been set to `false` **or** `content_id` has not been specified then this comparison will not occur. 
+
+### GEO whitelisting
+If `geo_whitelist` has been added to the DRM policy, when a license is requested the GEO location of the client's ip will be checked. If the country is in the whitelist the license request will be successful. If the country is not in the whitelist the request will be denied. E.g. if the DRM policy is `{ "geo_whitelist":[ "gbr", "deu" ] }` only users from the UK and Germany will be able to make successful license requests.
+
+### DRM session in policy
+
+It is possible to set in the policy used by VUDRM token information about the current DRM session. With this information we will make a request to make to an API of your choosing with an ID and any needed headers. You can set the request to be either a GET request or a POST request. In the case of a GET request we would make a GET request to the URL you specify with a query string parameter called `sessionId` set to ID the passed in the policy. E.g. a GET request to `https://some-url.com/valid?sessionId=abc123`. In the case of a POST request we would make a POST request to the URL you specify with the body of the request being the `sessionId` in JSON. E.g. a POST request to `https://some-url.com/valid` with the body being `{"sessionId":"abc123"}`. The result of the request we make will dictate whether or not a license is returned. If the result **is** a `200` we will serve a valid license back. If the result is **not** a `200` we will not return a valid license. 
```

Comment:
Rephrase: With this information we will make... etc (the word 'make' is duplicated in the sentence, can remove the second one)